cockatrice/src/game/player/player.cpp

```diff
@@ -997,7 +1002,7 @@ void Player::setShortcutsActive()
     while (counterIterator.hasNext()) {
         counterIterator.next().value()->setShortcutsActive();
     }
-
+    aIncrementAllCardCounters->setShortcut(shortcuts.getSingleShortcut("Player/aIncrementAllCardCounters"));
```

Comment:
Fyi, #6059 will fix a bug caused by using the wrong way of getting shortcuts. 
You might want to fix it on your end to be ahead of the curve.
```suggestion
    aIncrementAllCardCounters->setShortcuts(shortcuts.getShortcut("Player/aIncrementAllCardCounters"));
```